R/AddCardModule.R

```diff
@@ -140,7 +146,7 @@ add_card_button_srv <- function(id, reporter, card_fun) {
         card_fun_args_nams <- names(formals(card_fun))
         has_card_arg <- "card" %in% card_fun_args_nams
         has_comment_arg <- "comment" %in% card_fun_args_nams
-
+        has_label_arg <- "label" %in% card_fun_args_nams
```

Comment:
It is not used now. I think when have a label without a default value it will fail to eval `card <- try(do.call(card_fun, arg_list))`.

Reply:
removed.
and actually, it does not fail.

Reply:
not possible you test it as at the beginning of server function we have `checkmate::assert_subset(names(formals(card_fun)), c("card", "comment"), empty.ok = TRUE)`.:)

Reply:
Then we get expected by me error.

Reply:
Please add description of the new arg in the roxygen2 #' @param card_fun ...

Reply:
done.